src/basestyle/phosphor.less

```diff
@@ -168,15 +172,16 @@
 /*-----------------------------------------------------------------------------
 | TabBar
 |----------------------------------------------------------------------------*/
+
 .p-TabBar {
   overflow: visible;
   color: #777777;
-  font-size: 13px;
+  font-size: @jp-ui-font-size;
```

Comment:
Thought about that - the main difference is that things like `font-size` are actual CSS properties. Many/most of our less variables will have names like `@jp-component-css-property` so I would prefer to use a slightly modified naming convention here.